src/apps/material/material.tsx

```diff
@@ -207,11 +204,7 @@ const Material: React.FC<MaterialProps> = ({ wid }) => {
           cyData="material-reviews-disclosure"
         >
           <MaterialReviews
-            listOfReviews={
-              reviews as Array<
-                LibrariansReview | ExternalReview | InfomediaReview
-              >
-            }
+            pids={hasReview.map((review) => review.pid as Pid)}
```

Comment:
I think you should look into the `Work`type and see if you can add the `Pid`type there on the review. I think we should avoid type assertions as much as possible. Because with type assertions we risk the possibility of enforcing a wrong type to a value.